desktop/integration-test/openFile.test.ts

```diff
@@ -0,0 +1,39 @@
+// SPDX-FileCopyrightText: Copyright (C) 2023-2024 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)<lichtblick@bmwgroup.com>
+// SPDX-License-Identifier: MPL-2.0
+
+import path from "path";
+
+import { AppType, launchApp } from "./launchApp";
+
+describe("menus", () => {
```

Comment:
Update the name of the test please, it is "menus" and should be something like "open files"